lib/msal-node/src/response/ManagedIdentityTokenResponse.ts

```diff
@@ -28,6 +28,28 @@ export type ManagedIdentityTokenResponse = {
     token_type?: AuthenticationScheme;
 
     // error
-    message?: string; // will be converted to error
-    correlationId?: string; // will be converted to correlation_id
+
+    /*
+     * (Web/Function) App Service
+     * 500 errors can return this from all MI sources as well
```

Comment:
Not sure what this comment means. The error responses from different MI sources are similar for all the errors

Reply:
I'm not sure what _you_ mean. These are two separate comments. Does that that clarify it for you?